npm-debug.log

```diff
@@ -0,0 +1,23 @@
+0 info it worked if it ends with ok
```

Comment:
You have to clear the git cache if it got committed at any point: http://stackoverflow.com/questions/41863484/clear-git-local-cache